src/test/fuzz/addrman.cpp

```diff
@@ -285,7 +285,14 @@ FUZZ_TARGET(addrman, .init = initialize_addrman)
     auto max_pct = fuzzed_data_provider.ConsumeIntegralInRange<size_t>(0, 4096);
     auto filtered = fuzzed_data_provider.ConsumeBool();
     (void)const_addr_man.GetAddr(max_addresses, max_pct, network, filtered);
-    (void)const_addr_man.Select(fuzzed_data_provider.ConsumeBool(), network);
+
+    std::unordered_set<Network> nets;
+    LIMITED_WHILE(fuzzed_data_provider.ConsumeBool(), ALL_NETWORKS.size()) {
+        auto net{fuzzed_data_provider.PickValueInArray(ALL_NETWORKS)};
```

Comment:
> This still reads a bit odd, because a set shouldn't hold duplicate values, but the loop may produce duplicate values?

Yes, it may produce but would be "ignored" when trying to add. 

> Not sure what you are trying to achieve, but for (const auto& net : ALL_NETWORKS) { if (fdp.Bool()) nets.insert(net); } may be what you are looking for?

Yes, will address it.

Reply:
Done, thanks.